packages/GuaCaml/GuaCaml.0.02/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+name: "GuaCaml"
+version: "0.02"
```

Comment:
`name` and `version` are actually redundant, `opam` does replace them with the one inferred via the path on the `opam-repository` and, as you saw, emits a warning when they don't coincide. If you want to avoid the trouble, you can remove them from the `opam` file.

In the past we used to delete them, but this is no longer enforced, so feel free to keep them if you prefer.

Reply:
thank you for the explanation =)
I'll keep them